tempesta_fw/http_msg.c

```diff
@@ -1418,7 +1418,7 @@ tfw_h2_msg_rewrite_data(TfwHttpTransIter *mit, const TfwStr *str,
 			return -E2BIG;
 		}
 
-		memcpy_fast(mit->curr_ptr, c->data + c_off, n_copy);
+		tfw_cstrtolower(mit->curr_ptr, c->data + c_off, n_copy);
```

Comment:
With this change not only header names are converted to lowercase, but also header values. Such behaviour in unacceptable  and harmful.